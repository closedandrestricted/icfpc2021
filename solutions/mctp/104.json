{"vertices":[[42,49],[36,36],[48,57],[53,69],[27,72],[49,62],[36,58],[53,66],[54,64],[45,57],[58,80],[61,90],[55,71],[68,58],[63,76],[58,83],[57,78],[55,53],[54,52],[62,66],[59,64],[59,69],[44,55],[55,67],[56,57],[56,62],[42,48],[54,53],[35,59],[50,53],[26,70],[37,54],[40,46],[35,58],[39,30],[23,75]]}