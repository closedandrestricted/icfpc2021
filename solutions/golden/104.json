{"vertices":[[50,117],[95,105],[80,65],[63,98],[2,43],[76,83],[18,49],[69,60],[38,52],[28,45],[55,66],[58,76],[48,60],[28,68],[50,61],[56,69],[54,64],[29,33],[20,81],[44,52],[52,50],[50,55],[39,28],[50,49],[58,54],[43,58],[23,66],[47,37],[38,18],[65,47],[52,8],[72,36],[77,47],[77,66],[98,23],[100,69]]}